subxt/src/rpc/rpc.rs

```diff
@@ -31,16 +31,19 @@
 //! # }
 //! ```
 
+use std::sync::Arc;
+
+use codec::{Decode, Encode};
+use frame_metadata::RuntimeMetadataPrefixed;
+use serde::Serialize;
+
+use crate::{error::Error, utils::PhantomDataSendSync, Config, Metadata};
```

Comment:
If possible we should add some `cargo fmt` rules to control how imports are grouped etc, just so it's standard everywhere

Reply:
yes, that would be good